lib/ruby_smb/client.rb

```diff
@@ -306,18 +307,12 @@ def initialize(dispatcher, smb1: true, smb2: true, smb3: true, username:, passwo
       # SMB 3.x options
       @session_encrypt_data = always_encrypt
 
-      negotiate_version_flag = 0x02000000
-      flags = Net::NTLM::Client::DEFAULT_FLAGS |
-        Net::NTLM::FLAGS[:TARGET_INFO] |
-        negotiate_version_flag ^
-        Net::NTLM::FLAGS[:OEM]
-
       @ntlm_client = Net::NTLM::Client.new(
         @username,
         @password,
         workstation: @local_workstation,
         domain: @domain,
-        flags: flags
+        ntlm_flags: ntlm_flags
```

Comment:
I might be missing something here, but I cannot find the `ntlm_flags` key word argument being used in the `Net::NTLM::Client` class [constructor](https://github.com/WinRb/rubyntlm/blob/master/lib/net/ntlm/client.rb#L21-L27).